command/cp.go

```diff
@@ -109,6 +110,9 @@ Examples:
 		 
 	23. Download the specific version of a remote object to working directory
 		 > s5cmd {{.HelpName}} --version-id VERSION_ID s3://bucket/prefix/object .
+
+	23. Pass arbitrary metadata to the object during upload or copy 
+		 > s5cmd {{.HelpName}} --metadata "foo=bar" --metadata "fizz=buzz" localfile s3://bucket/prefix/object 
```

Comment:
could you provide real-life metadata examples?

Reply:
Refactored here: [0764892](https://github.com/peak/s5cmd/pull/621/commits/07648927364c8d2fab944d27b720458ad1007150)